app/scripts/models/reliers/relier.js

```diff
@@ -102,6 +102,12 @@ define(function (require, exports, module) {
           } else {
             self.importSearchParam('email');
           }
+
+          self.importSearchParam('migration');
+          if (self.has('migration')) {
```

Comment:
Specifically, I don't like the idea of "be liberal in what you accept" when it comes to security-related inputs like `scope` and `redirect_uri`, so for consistency, we may has well avoid normalization on all params here.